notion/collection.py

```diff
@@ -360,6 +360,7 @@ def __init__(
         sort=[],
         calendar_by="",
         group_by="",
+        limit=100
```

Comment:
Is it possible to increase the default limit for `CollectionQuery`? The current value doesn't fetch all rows from my database with ~200 items. I didn't run into any API errors with a higher value (100000) during my short test.

Reply:
I agree, I tried same value for limit and it runs without an issue (https://charts.mathix.ninja/).

But it would be nice to have some sort of pagination, as after 100000 documents the same issue will occur.

Reply:
Have you tried: limit=-1? How is it working with 10000+ ?